build/ci.go

```diff
@@ -287,7 +288,31 @@ func doTest(cmdline []string) {
 	if *vet {
 		build.MustRun(goTool("vet", packages...))
 	}
+	if *misspell {
+		// Ensure the spellchecker is available
+		build.MustRun(goTool("get", "github.com/client9/misspell/cmd/misspell"))
 
+		// Windows (AppVeyor) chokes on long argument lists, check packages individualy
```

Comment:
The code is exactly the same without it too. The only difference is callign spellcheck for all files in one huge batch or separately for each package. See first commit for the initial version.